erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md

```diff
@@ -0,0 +1,31 @@
+<!-- add-breadcrumbs -->
+# Immutable Ledger In ERPNext
```

Comment:
add a note in Journal Entry, Stock Entry, Purchase Receipt, Delivery Note, wherever this applies. Something like

Note: From v13, rules for stock and accounting entries have changed with Immutable ledger. Backdated entries are no longer allowed and Journal Entries are cancelled by using a reverse entry. [Learn more here](/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext).